build/repo.targets

```diff
@@ -19,21 +19,20 @@
     <Exec Command="npm install --no-optional" WorkingDirectory="$(RepositoryRoot)clients/ts/FunctionalTests" />
     <Exec Command="npm install --no-optional" WorkingDirectory="$(RepositoryRoot)clients/ts/signalr" />
     <Exec Command="npm install --no-optional" WorkingDirectory="$(RepositoryRoot)clients/ts/signalr-protocol-msgpack" />
-    <Exec Command="npm install --no-optional" WorkingDirectory="$(RepositoryRoot)clients/ts/webdriver-tap-runner" />
   </Target>
 
   <PropertyGroup>
     <TestDependsOn>$(TestDependsOn);RunTSClientNodeTests;RunBrowserTests</TestDependsOn>
   </PropertyGroup>
 
   <Target Name="RunTSClientNodeTests">
-    <Message Text="Running TypeScript client Node tests" Importance="high" />
+    <Message Text="Running JavaScript client Node tests" Importance="high" />
```

Comment:
I got nitpicky. The client is **written** in TypeScript but it runs on a JavaScript platform. I was waffling on the name earlier so I used TypeScript here but I've since been consolidating on calling it the "JavaScript client"